### ISSUE TITLE
openSUSE Pmw tests fail again

### ISSUE THREAD

**Issue by jayvdb** [Nuitka/Nuitka#810]
Last issue was https://github.com/Nuitka/Nuitka/issues/420
Current trace on x86_64 & i586 / py27 & py38 is
```
[ 4005s] Comparing output of 'PmwUsing.py' using '/usr/bin/python3' with flags silent, expect_success, standalone, remove_output, ignore_infos, plugin_enable:pmw-freezer ...
[ 4245s] --- /usr/bin/python3 (stderr)
[ 4245s] +++ nuitka (stderr)
[ 4245s] 
[ 4245s] @@ -1 +1,5 @@
[ 4245s] 
[ 4245s] +Traceback (most recent call last):
[ 4245s] +  File "/home/abuild/rpmbuild/BUILD/Nuitka-0.6.8.4/tests/standalone/PmwUsing.dist/PmwUsing.py", line 20, in <module>
[ 4245s] +  File "/home/abuild/rpmbuild/BUILD/Nuitka-0.6.8.4/tests/standalone/PmwUsing.dist/Pmw/__init__.py", line 2, in <module>
[ 4245s] +ImportError: cannot import name 'PmwColor' from 'Pmw' ( >> /home/abuild/rpmbuild/BUILD/Nuitka-0.6.8.4/tests/standalone/PmwUsing.dist/Pmw/__init__.py)
[ 4245s]  
[ 4245s] Exit codes 0 (CPython) != 1 (Nuitka)
[ 4245s] Error, outputs differed.
[ 4245s] Listing of dist folder:
[ 4245s] -rw-r--r-- 1 abuild abuild 1038 Jun 17 13:01 PmwUsing.py
[ 4245s] Error exit! 1
[ 4245s] error: Bad exit status from /var/tmp/rpm-tmp.BqooLg (%check)
[ 4245s] 
[ 4245s] 
[ 4245s] RPM build errors:
[ 4245s]     Bad exit status from /var/tmp/rpm-tmp.BqooLg (%check)
[ 4245s] ### VM INTERACTION START ###
[ 4245s] --- /usr/bin/python3 (stderr)
[ 4245s] +++ nuitka (stderr)
[ 4245s] 
[ 4245s] @@ -1 +1,5 @@
[ 4245s] 
[ 4245s] +Traceback (most recent call last):
[ 4245s] +  File "/home/abuild/rpmbuild/BUILD/Nuitka-0.6.8.4/tests/standalone/PmwUsing.dist/PmwUsing.py", line 20, in <module>
[ 4245s] +  File "/home/abuild/rpmbuild/BUILD/Nuitka-0.6.8.4/tests/standalone/PmwUsing.dist/Pmw/__init__.py", line 2, in <module>
[ 4245s] +ImportError: cannot import name 'PmwColor' from 'Pmw' ( >> /home/abuild/rpmbuild/BUILD/Nuitka-0.6.8.4/tests/standalone/PmwUsing.dist/Pmw/__init__.py)
[ 4245s]  
[ 4245s] Exit codes 0 (CPython) != 1 (Nuitka)
[ 4245s] Error, outputs differed.
[ 4245s] Listing of dist folder:
[ 4245s] -rw-r--r-- 1 abuild abuild 1038 Jun 17 13:01 PmwUsing.py
[ 4245s] Error exit! 1
[ 4245s] error: Bad exit status from /var/tmp/rpm-tmp.BqooLg (%check)
```
https://build.opensuse.org/package/rdiff/home:jayvdb:py-bridge/python-Nuitka?linkrev=base&rev=44

**Comment by kayhayen** [comment]
I managed to reproduce this one too. Not sure what's going on there, but this should be a regression, so likely I can just bisect it.

**Comment by kayhayen** [comment]
Fix is on factory, I messed up when changing plugin command line options, confused by negated defaulted values used.

**Comment by kayhayen** [comment]
Also released as 0.6.9 already.

**Comment by jayvdb** [comment]
Confirmed fixed.
